### PR TITLE
Migrate tests-legacy tests to namespace LegacyTests

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migrate tests-legacy tests to namespace LegacyTests
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10733
| How to test?  | Travis tests should be green

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11529)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
No need QA, wiating for someone else to approve and merge :)

File: tests-legacy/Unit/Core/Webservice/ServerRequirementsCheckerTest.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-namespace Tests\Unit\Core\Webservice;
+namespace LegacyLegacyTests\Unit\Core\Webservice;
```
Comment (jolelievre) [inline tests-legacy/Unit/Core/Webservice/ServerRequirementsCheckerTest.php]: `LegacyTests` too much legacy!

**Review by jolelievre — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by jolelievre — CHANGES_REQUESTED** [review]
Careful, you integrated rtl css files in the PR which have no link with this PR
And shouldn't be commit anyway since they are generated on install

**Comment by matks** [comment]
My bad 😢 
Commit https://github.com/PrestaShop/PrestaShop/pull/11529/commits/0ea626e3723c602fdc99a1682e13de73a8737faf fixed by push-force

**Review by jolelievre — APPROVED** [review]

**Pull request merged**